pythoncz/models/github.py

```diff
@@ -81,6 +71,27 @@ def _get_issues_for_org(session, org_name):
         )
 
 
+def _request_api(session, query, variables):
+    res = session.post('https://api.github.com/graphql', json={
+        'query': query,
+        'variables': variables,
+    })
+    try:
+        json = res.json()
+    except ValueError as e:
+        res.raise_for_status()
+        raise ValueError(
+            'Unexpected structure of the GitHub API response: {}'.format(e)
+        )
+    else:
```

Comment:
No strong opinions here either. Slightly inclined to remove it, as I would change it in this case:

```python
def foo():
    if whatnot():
        ...
        return 123
    else:
        ...
        return 563
```

To:


```python
def foo():
    if whatnot():
        ...
        return 123
    ...
    return 563
```